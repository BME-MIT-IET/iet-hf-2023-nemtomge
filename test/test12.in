loadmap map.json 
start V1 V2 
set Scientist V1 PL1 
set Scientist V2 PL1 
give Gear V1 Gloves 
give Gear V1 Axe 
give Gear V1 Bag 
give Gear V2 Coat 
give Active V2 Stun 3 
rob V1 V2 
1
exit
