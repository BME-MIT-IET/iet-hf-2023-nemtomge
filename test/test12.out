Map from file map.json loaded
The game has been started. Players are: V1 V2
Scientist V1 set on field PL1 
Scientist V2 set on field PL1 
Scientist V1 has been given gear Gloves 
Scientist V1 has been given gear Axe 
Scientist V1 has been given gear Bag 
Scientist V2 has been given gear Coat 
Scientist V2 has been given active agent Stun time: 3 
Choose one to drop:
0 - Cancel
1 - Gloves(3)
2 - Axe(1)
3 - Bag
Scientist V1 has robbed V2
